python/postgres_etl/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.ruff]
+
+# Set same as Black.
+line-length = 100
+indent-width = 4
+
+[tool.ruff.lint]
+select = ["I"]
+ignore = ["E501"]
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
```

Comment:
If you add this in the `pyproject.toml` then you won't need to add the `@pytest.mark.asyncio` decorator to tests.

```toml
[tool.pytest.ini_options]
asyncio_mode = "auto"
```